fivetran/connectors.tf

```diff
@@ -54,3 +52,31 @@ resource "fivetran_connector" "github" {
     pat          = each.value.pat
   }
 }
+
+module "lambda_connector" {
+  for_each = { for lambda in var.sources_lambda : join("-", compact([lambda.service_name, lambda.project, lambda.environment])) => lambda }
```

Comment:
Will the combination of service/project/environment result in duplicated values for a multi-regions setup? If so, should the region code be included in the unique key?

Reply:
Refer to this comment -> https://github.com/dbl-works/terraform/pull/137#issuecomment-1281859748

Since we only can create lambda in one region, I think this is OK?